### PR TITLE
Do not remove the accels from the module list.

### PR DESCRIPTION
**Pull request by TurboGit**
We want to only disconnect the accels from the global accelerators.
This fixes an issue where after closing an IOP and opening it again
the accels where not working.

### CODE REVIEW

**Comment by LebedevRI** [comment]
This is about those accels created with `dt_accel_connect_iop()` - e.g `[` and `]` in flip.c

**Comment by LebedevRI** [comment]
@TurboGit i do acknowledge that the bug exists.
But the code does not fix it for me.
Checked with `[` and `]` in `orientation` iop.

Please double-check :)

**Comment by TurboGit** [comment]
@LebedevRI hum... strange it works on my side.

What's your testing procedure?

Here is what I did:

- open a picture on darkroom
- select the orientation module and played with [ and ], the picture is rotated
- I have selected another iop (crop or exposure)
- then back to orientation and playing with [ and ] is still working

This is the last step that was not working before as nothing happened. Maybe there is a case where this is still not working... So please let me know how to reproduce. Thanks.

**Comment by LebedevRI** [comment]
> What's your testing procedure?

- open a picture on darkroom
- (basecurve iop is selected)
- select the orientation module
- `[` and `]` are not working
- selected another iop (grad density)
- back to orientation
- `[` and `]` is still not working

If i switch to dr when orientation module is selected, then `[` and `]` are working until i switch away to other module

TLDR: i see no difference compared to git master...

**Comment by TurboGit** [comment]
Really strange I have tested again... all is working fine for me with the scenario you described :(

I have tested again from current master and I can reproduce the issue. When I build with my proposed path all is working fine on my side.

I have double/triple checked by removing my install and build directory completely.

**Comment by LebedevRI** [comment]
Did some more digging.
```
dt_accel_connect_locals_iop: "flip" "<Darktable>/image operations/flip/rotate 90 degrees CCW"
dt_accel_connect_locals_iop: "flip" "<Darktable>/image operations/flip/rotate 90 degrees CW"
dt_accel_disconnect_locals_iop: "flip" "<Darktable>/image operations/flip/rotate 90 degrees CCW"
dt_accel_disconnect_locals_iop: "flip" "<Darktable>/image operations/flip/rotate 90 degrees CW"
dt_accel_connect_locals_iop: "flip" "<Darktable>/image operations/flip/rotate 90 degrees CCW"
dt_accel_connect_locals_iop: "flip" "<Darktable>/image operations/flip/rotate 90 degrees CW"
```
So local accels are indeed reconnected, even weirder.

**Comment by TurboGit** [comment]
Really strange indeed! So in your case the local accels are never working. Looks like another issue? Maybe due to the WM used?

**Comment by LebedevRI** [comment]
No, like i said in https://github.com/darktable-org/darktable/pull/1192#issuecomment-209475027, they work if `orientation iop` is opened when entering `darkroom`.
Once i close it, no matter what i do, they no longer work.

**Comment by TurboGit** [comment]
Ok I tried again and again. No matter how if the orientation iop is open or closed it works for me. I have also tested with landscape and portrait picture, it works for me. I have also tested with picture with orientation already in the history or not and the iop open or closed when entering the darkroom. For me it still works! I'm out of idea.

Without my patch it fails most of the time, so this patch definitely makes things far better on my side. Maybe an issue with the Gtk+ version used or the WM? I'm using GNOME Shell with 3.20.2-1.

**Comment by LebedevRI** [comment]
Gtk+-3.18, kde

**Comment by TurboGit** [comment]
kde may be the explanation here... not sure. if it is not worth on your side and better on mine I'll probably merge this, we will have more testers.

**Comment by TurboGit** [comment]
Let's merge this to have more feedbacks. I'll try to go to the bottom of this!
